mtp_noms_ops/apps/security/forms.py

```diff
@@ -203,10 +203,10 @@ class SenderGroupedForm(SecurityForm):
 
     prisoner_count_0 = forms.IntegerField(label=_('Number of prisoners'), required=False, min_value=1)
     prisoner_count_1 = forms.IntegerField(label=_('Maximum prisoners sent to'), required=False, min_value=1)
-    credit_count_0 = forms.IntegerField(label=_('Minimum credits sent'), required=False, min_value=1)
-    credit_count_1 = forms.IntegerField(label=_('Maximum credits sent'), required=False, min_value=1)
-    credit_total_0 = forms.IntegerField(label=_('Minimum total sent'), required=False)
-    credit_total_1 = forms.IntegerField(label=_('Maximum total sent'), required=False)
+    credit_count_0 = forms.IntegerField(label=_('Min'), required=False, min_value=1)
```

Comment:
the problem with having labels that are incomplete is the error message becomes
<img width="243" alt="screen shot 2016-07-11 at 15 42 26" src="https://cloud.githubusercontent.com/assets/911238/16734634/257fe7fc-477e-11e6-8311-4782f74702f8.png">